ext/com_dotnet/com_wrapper.c

```diff
@@ -305,7 +307,7 @@ static HRESULT STDMETHODCALLTYPE disp_invokeex(
 				VariantInit(pvarRes);
 				php_com_variant_from_zval(pvarRes, retval, COMG(code_page));
 			}
-			zval_ptr_dtor(retval);
+			// zval_ptr_dtor(retval); // TODO needed for function calls?
```

Comment:
Yes I believe this is needed for function calls

Reply:
See

> Note that this doesn't fix calling methods, which is broken since 2005, because DISPATCH_METHOD|DISPATCH_PROPERTYGET is passed as flag, and the latter is always prioritized in disp_invokeex() (what is likely good for regular COM objects).

I'm planning to fix this in a follow-up PR. Thus I left the comment to not forget about destroying.

Reply:
Oh, I overlooked this. This looks ok to me then